03 Misc/08 DefinePlugin advanced/src/totalScoreComponent.jsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
```

Comment:
Same here i think I would remove this component and only focus on the student list